source/components/other-utils.md

```diff
@@ -50,6 +50,23 @@ window.addEventListener(
 )
 ```
 
+Vue SFV example:
+```
+<template>
+<q-input @keyup="save(submission)" v-model="submission"  />
+</template
+<script>
+export default {
+ import { debounce } from 'quasar'
+ methods: {    
+    save: debounce(function(field, value) {
```

Comment:
you're calling the "save" method with one parameter only (the value), but it's definition has two parameters, out of which the first one is the "field"